`tests/parsers/ebml` missing from sdist for 0.5.0

The sdist archive for 0.5.0 is missing the `tests/parsers/ebml` directory, which leads to test failures:

```pytb
____________________________________________________ EBMLTestCase.test_parse_full _____________________________________________________

self = <test_parsers.EBMLTestCase testMethod=test_parse_full>

    def setUp(self):
        self.stream = io.open(os.path.join(TEST_DIR, "test1.mkv"), "rb")
>       with io.open(os.path.join(EBML_VALIDATION_DIR, "test1.mkv.yml"), "r") as yml:
E       FileNotFoundError: [Errno 2] No such file or directory: '/tmp/portage/dev-python/enzyme-0.5.0/work/enzyme-0.5.0/tests/parsers/e
bml/test1.mkv.yml'

self       = <test_parsers.EBMLTestCase testMethod=test_parse_full>

tests/test_parsers.py:27: FileNotFoundError
```

(and so on)

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,0 +1,1 @@
+recursive-include tests *.yml

